perf-regression/polaris/run_ior.qsub

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+#PBS -l select=1:system=polaris
+#PBS -l place=scatter
+#PBS -l walltime=0:10:00
+#PBS -l filesystems=home:grand
```

Comment:
Oh, actually the "grand" filesystem requirement can be removed here as well.  I don't think this script actually uses that file system.